test/spectests/spectests.cpp

```diff
@@ -15,6 +16,31 @@ namespace
 constexpr auto JsonExtension = ".json";
 constexpr auto UnnamedModule = "_unnamed";
 
+/* wat2wasm
```

Comment:
Can you add these links here (you've listed):
WebAssembly/spec#1111
https://github.com/WebAssembly/spec/blob/e3a3c12f51221daaa901691e674d7ff990029122/interpreter/host/spectest.ml#L33
https://github.com/WebAssembly/spec/blob/f85f6e6b70fff9ee0f6ba21ebcdad2183bd67874/test/harness/sync_index.js#L79

Reply:
Added in https://github.com/wasmx/fizzy/pull/215